eslzArm/managementGroupTemplates/policyAssignments/DINE-MDEndpointsPolicyAssignment.json

```
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "topLevelManagementGroupPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Provide the ESLZ company prefix to the intermediate root management group containing the policy definitions."
+            }
+        },
+        "enforcementMode": {
+            "type": "string",
+            "allowedValues": [
+                "Default",
+                "DoNotEnforce"
+            ],
+            "defaultValue": "Default"
+        },
+        "enableMDEndpoints": {
+            "type": "string",
+            "allowedValues": [
+                "Disabled",
+                "DeployIfNotExists",
+                "AuditIfNotExists"
+            ],
+            "defaultValue": "DeployIfNotExists"            
+        }
+    },
+    "variables": {
+        "policyDefinitions": {
+            "deployMDEndpoints": "/providers/Microsoft.Authorization/policySetDefinitions/e20d08c5-6d64-656d-6465-ce9e37fd0ebc"
+        },
+        "policyAssignmentNames": {
+            "azureSecurityMDE": "Deploy-MDEndpoints",
+            "description": "Deploy Microsoft Defender for Endpoint agent on applicable images.",
+            "displayName": "[[Preview]: Deploy Microsoft Defender for Endpoint agent"
```

```suggestion
            "displayName": "[Preview]: Deploy Microsoft Defender for Endpoint agent"
```
Dont think you need this here, only in policy rules etc...